src-docs/src/views/i18n/context.js

```diff
@@ -28,12 +28,13 @@ const mappings = {
   },
 };
 
-const ContextConsumer = () => {
+const ContextConsumer = (props) => {
   const [name, setName] = useState('');
+  const { language } = props;
```

Comment:
code style nit: I would just destructure in place

```suggestion
const ContextConsumer = ({ language }) => {
  const [name, setName] = useState('');
```

Reply:
😆 I had it destructured in place at first, then refactored and went more explicit with the internal declaration. I'll flip it back!